__tests__/app.test.js

```diff
@@ -312,3 +313,48 @@ describe("GET /api/users", () => {
       });
   });
 });
+describe("GET /api/articles(queries)", () => {
+  test("should return articles by the topic value specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles[0]).toEqual({
+          author: "rogersop",
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          article_id: 5,
+          topic: "cats",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: 2,
+        });
+      });
+  });
+  test("should return articles sorted by any valid column specified in the query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("votes", { descending: true });
+      });
+  });
+  test("should return articles sorted by author by topic mitch ASC specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=author&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("author", { descending: false });
+      });
+  });
+  test("should return error if bad query value", () => {
+    return request(app)
+      .get("/api/articles?sort_by=bad")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
```

Comment:
You need 2 more sad path tests: 
 400 Bad request (order_by=bananas) 
404 Not found (topic=bananas) 

These are different because the topic _could_ one day be bananas however it's unrealistic to ever be able to order_by bananas 👍